lib/command_cursor.js

```diff
@@ -55,62 +52,92 @@ const MongoDBNamespace = require('./utils').MongoDBNamespace;
  * @fires CommandCursor#readable
  * @return {CommandCursor} an CommandCursor instance.
  */
-var CommandCursor = function(topology, ns, cmd, options) {
-  CoreCursor.apply(this, Array.prototype.slice.call(arguments, 0));
-  var state = CommandCursor.INIT;
```

Comment:
Why does this go away?

Reply:
Since this class is now property `extend`ing the `Cursor` class, all of this is handled in the super classes' constructor.